tests/integration/conftest.py

```diff
@@ -0,0 +1,17 @@
+import pytest
+
+
+def pytest_addoption(parser):
```

Comment:
In order for this to work in CI, you'll apparently need to put the file at tests/conftest.py instead.  I think what happens is that pytest finds `testpaths = tests` in tox.ini and then only processes tests/conftest.py prior to handling command-line options.